DQMOffline/Configuration/python/DQMOfflineHeavyIons_cff.py

```diff
@@ -15,6 +15,12 @@
 from DQM.BeamMonitor.AlcaBeamMonitorHeavyIons_cff import *
 from DQMOffline.L1Trigger.L1TriggerDqmOffline_cff import *
 
+### add for electron                                                                                                                                                                
+from DQMOffline.EGamma.electronOfflineClientSequence_cff import *
+from Validation.RecoEgamma.electronValidationSequence_cff import *
+from Validation.RecoEgamma.electronPostValidationSequence_cff import *
```

Comment:
@BetterWang , @deguio  - worried about importing Validation* into DQMOffline*. Shouldn't these imports and then replace statements below be instead in the corresponding ValidationHeavyIons_cff?